docs/fassets/guides/deploy-fassets-bots.mdx

```diff
@@ -0,0 +1,215 @@
+|          `FTestDOGE` | [`0x353D20c6eB0C7bfcE3B828665C992DE995eF67bd`](https://coston-explorer.flare.network/token/0x353D20c6eB0C7bfcE3B828665C992DE995eF67bd)   | The FAsset-wrapped TestDOGE token, ready for use on Songbird Testnet Coston.                                                             |
+
+## Setting Up
+
+Follow these steps to set up the FAssets agent using Docker:
+
+1. Clone the repository in your Linux server:
+
+   ```bash
+   git clone https://gitlab.com/flarenetwork/fasset-bots-deploy
+   cd fasset-bots-deploy
+   ```
+
+2. Set up the environment file:
+
+   ```bash
+   cp .env.template .env
+   ```
+
+3. Set up the configuration file:
+
+   ```bash
+   cp config.json.template config.json
+   ```
+
+4. Login into GitHub Container Registry:
+
+   ```bash
+   docker login ghcr.io
+   ```
+
+5. Get the latest docker containers from the registry:
+
+   ```bash
+   docker compose pull
+   ```
+
+### Setting Environment Variables
+
+To begin using the FAssets Docker infrastructure, you must configure the settings in the `.env` environment file.
+
+- Make sure to set the machine address in the `.env` file using the `MACHINE_ADDRESS` variable.
+- Set the database password `FASSET_DB_PASSWORD`.
+
+:::warning
+Once the database is created, the password will not change if you modify it in the `.env` file.
+:::
+
+#### Optional Settings
+
+You can configure several optional settings in the `.env` environment file.
+
+- Choose the profiles of the FAssets agent to run in `COMPOSE_PROFILES`, which by default include the FAssets agent bot, the agent user interface, and the liquidator bot.
+- The frontend interface operates on the port designated as `FRONTEND_PORT`, which is set to `3000` by default.
+- The link for the agent frontend user interface, referred to as `FRONTEND_URL`, is empty by default.
+- The password for the agent frontend user interface is `FRONTEND_PASSWORD`.
+- The backend port is `BACKEND_PORT`, which defaults to `4000`.
+- The backend URL is `BACKEND_URL`, which defaults to `/fasset-backend`.
+
+## Setting up Secret Keys
+
+The FAsset agents require multiple keys for the Flare and underlying network chains, which must be generated to make the agent operational.
+
+1. Create a new management wallet or use an existing one, preferably a cold wallet, as the management address for your agent.
+2. Generate the secrets using this command by replacing the `MANAGEMENT_WALLET_ADDRESS` with your management wallet address:
+
+   ```bash
+   ./generateSecrets.sh MANAGEMENT_WALLET_ADDRESS
+   ```
+
+3. Copy the `secrets.new.json` to `secrets.json`:
+
+   ```bash
+   cp secrets.new.json secrets.json
+   ```
+
+4. Prevent other users from reading the secrets.json file:
```

Comment:
Can we instead say something like "give user read and write rights to the `secrets.json` file"

Reply:
Done